apps/fishing-map/features/map/map.hooks.ts

```diff
@@ -54,15 +54,28 @@ import {
   fetchFishingActivityInteractionThunk,
   fetchViirsInteractionThunk,
   selectViirsInteractionStatus,
-  ExtendedViirsFeature,
+  ApiViirsStats,
 } from './map.slice'
 import useViewport from './map-viewport.hooks'
 import { useMapAndSourcesLoaded } from './map-features.hooks'
 
 export const SUBLAYER_INTERACTION_TYPES_WITH_VESSEL_INTERACTION = [
   'fishing-effort',
   'presence-detail',
+  'viirs-match',
 ]
+export const SUBLAYER_INTERACTION_TYPES_WITH_VIIRS_INTERACTION = ['viirs']
```

Comment:
Do we actually plan to keep that type of interaction? (do we plan to keep VIIRS layer without radiance filter etc?)

Reply:
Nop, this is deprecated and once the prototype is ready for production we'll remove this. Added a comment so we don't forget it